library/core/registry/src/main/java/org/quiltmc/qsl/registry/api/dynamic/DynamicMetaRegistry.java

```diff
@@ -48,8 +48,8 @@ public final class DynamicMetaRegistry {
 	 * @param entryCodec the codec used to deserialize entries from datapacks
 	 * @throws IllegalStateException if this registry of registries already got frozen
 	 */
-	public static <E> void register(RegistryKey<? extends Registry<E>> key, Codec<E> entryCodec) {
-		DynamicMetaRegistryImpl.register(key, entryCodec);
+	public static <E> void register(RegistryKey<? extends Registry<E>> key, Codec<E> entryCodec, DynamicRegistryFlag... flags) {
```

Comment:
This change is not binary compatible, we may want to add an overload for the flagless version

Reply:
Fixed as of latest commit. Thanks for the catch, didn't know varargs would cause binary incompatibility like that!